tests/functional/docs/test_help_output.py

```diff
@@ -482,3 +483,11 @@ def test_service_help_command_has_note(self):
         self.driver.main(['s3api', 'get-object', 'help'])
         self.assert_not_contains('outfile <value>')
         self.assert_contains('<outfile>')
+
+
+# Use this test class for "help" cases that require the default renderer
+# (i.e. renderer from get_render()) instead of a mocked version.
+class TestHelpOutputDefaultRenderer:
```

Comment:
To fix this test, I got something like this working:
```diff
diff --git a/tests/functional/docs/test_help_output.py b/tests/functional/docs/test_help_output.py
index 5eaab8121..2af5fe7f5 100644
--- a/tests/functional/docs/test_help_output.py
+++ b/tests/functional/docs/test_help_output.py
@@ -22,11 +22,12 @@ at the man output, we look one step before at the generated rst output
 
 """
 import io
+import os
 
-from awscli.testutils import aws
 from awscli.testutils import BaseAWSHelpOutputTest
 from awscli.testutils import FileCreator
 from awscli.testutils import mock
+from tests import CLIRunner
 
 from awscli.compat import six
 from awscli.alias import AliasLoader
@@ -485,5 +486,15 @@ class TestStreamingOutputHelp(BaseAWSHelpOutputTest):
 # (i.e. renderer from get_render()) instead of a mocked version.
 class TestHelpOutputDefaultRenderer:
     def test_line_lengths_do_not_break_create_launch_template_version_cmd(self):
-        result = aws('ec2 create-launch-template-version help')
+        runner = CLIRunner()
+
+        # Add the PATH to the environment variables so that that posix help
+        # renderers can find either the groff or mandoc executables required to
+        # render the help pages for posix environments
+        if 'PATH' in os.environ:
+            runner.env['PATH'] = os.environ['PATH']
+
+        result = runner.run(
+            ['ec2', 'create-launch-template-version', 'help']
+        )
         assert 'exceeds the line-length-limit' not in result.stderr

```
From v1 to v2, we converted similar functional tests to use the `CLIRunner`. For example, this [test](https://github.com/aws/aws-cli/blob/v2/tests/functional/kinesis/test_remove_operations.py#L17). Using the runner should get around not find the `aws` executable on our path as it does not spawn a subprocess to run the CLI; it invokes the CLI driver explicitly in-process.